examples/astro-forms-demo/src/components/ContactsPhone.tsx

```diff
@@ -0,0 +1,68 @@
+import { type ContactsPhoneProps } from '@wix/headless-forms/react';
+import {
+  quickStartViewerPlugins,
+  RicosViewer,
+  type RichContent,
+} from '@wix/ricos';
+import '@wix/ricos/css/all-plugins-viewer.css';
+
+const ContactsPhone = ({
+  id,
+  value,
+  label,
+  showLabel,
+  placeholder,
+  description,
+  required,
+  readOnly,
+  allowedCountryCodes,
```

Comment:
I think we should provide more data, like:
[
   { 'LTU', 'Lithuania', '370' }, 
   { 'LV', 'Latvia', '371' }, 
 ]
 
 maybe even the flag image.
 also it could have "selected" property.
    { 'LV', 'Latvia', '371', selected: true }, 
```suggestion
  allowedCountryCodes,
```